src/app/components/engangsstonad/steps/Engangsstonad.step3.js

```diff
@@ -77,7 +79,22 @@ export class Step3 extends Component {
 	}
 
 	// eslint-disable-next-line class-methods-use-this, no-unused-vars
-	nextTwelveMonthsValueChange($e, stages, expandedStage) {}
+	nextTwelveMonthsValueChange($e, stages) {
+		stages.forEach((stage) => {
```

Comment:
Trenger vi denne, eller kan logikken her legges direkte på komponenten? Der settes vel stage?

Reply:
Isåfall må vi vel på en eller annen måte klare å få `TransformingRadioGroupCollection`/`TransformingRadioGroup`-komponentene til å mappe attributter fra parent (`Engangsstonad.step3`), og da må nok ting skrives om litt siden vi ikke vil ha stages in i redux-store men bare verdiene som settes, som er grunnen til at vi trenger den logikken her nå. 

Mulig vil vi i en sånn omskrivning vil få en utfordring med primitiver som ikke vil oppdateres i parent, men kanskje det kan løses noe wrapping. Generelt er jeg enig i at de komponentene har forbedringspotensiale, men tenker det er utenfor scope her.